angular/core/components/proposal_form/proposal_form.coffee

```diff
@@ -22,3 +25,10 @@ angular.module('loomioApp').factory 'ProposalForm', ->
 
     KeyEventService.submitOnEnter $scope
     MentionService.applyMentions $scope, $scope.proposal
+
+    $scope.$on 'disableAttachmentForm', -> $scope.submitIsDisabled = true
```

Comment:
Could we move this into AttachmentService.coffee rather than duplicating it?

```
listenForAttachments: (scope, model) ->
  scope.$on 'disableAttachmentForm', ->
  scope.$on 'enableAttachmentForm', -> 
  scope.$on 'attachmentRemoved', ->
    ids = model.newAttachmentIds
```
```
# proposal_form.coffee
AttachmentService.listenForAttachments($scope, $scope.proposal)
```